manifests/charts/ai-gateway-helm/values.yaml

```diff
@@ -196,12 +196,18 @@ controller:
   # resources (configuration) is big. Defaults to 4MB.
   # maxRecvMsgSize: "4194304"
 
-  # mcpSessionEncryptionSeed is an arbitrary string seed used to derive the MCP session encryption key.
-  #	Do not include commas as they are used as separators. You can optionally pass "fallback" seed after the first one to allow for key rotation.
-  #	For example: "new-seed,old-seed-for-fallback". The fallback seed is only used for decryption.
-  #
-  # This value should be set to a secure random string in production environments instead of the default below.
-  mcpSessionEncryptionSeed: "default-insecure-seed"
```

Comment:
This could be a breaking change, but we're in version `0.4` and the current Deployment template was not reading this value (!) anyway, so I think it is OK to just "move" this setting.